.github/workflows/release.yaml

```diff
@@ -0,0 +1,19 @@
+name: Release
```

Comment:
taken from the [instructions](https://github.com/olafurpg/sbt-ci-release#github-actions) only the branch was adjusted.